config/nimbus.yaml

```diff
@@ -101,6 +101,33 @@ features:
         value: { "enabled": true }
       - channel: production
         value: { "enabled": true }
+  landing-page-redesign:
+    description: Landing page redesign
```

Comment:
I wonder if this might be confusing later, could we give it a more unique name? Do we ever remove these?